swap_meet/vendor.py

```diff
@@ -1,2 +1,69 @@
+from .item import Item
+
+
 class Vendor:
-    pass
+    def __init__(self, inventory=None):
+        self.inventory = inventory if inventory is not None else []
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+        else:
+            self.inventory.remove(item)
+            return item
+
+    def get_by_category(self, category):
+        result = []
+        for item in self.inventory:
+            if item.category == category:
+                result.append(item)
+        return result
+
+    def swap_items(self, vendor_friend, my_item, their_item):
+        if my_item not in self.inventory or their_item not in vendor_friend.inventory:
+            return False
+        self.remove(my_item)
+        vendor_friend.add(my_item)
+        vendor_friend.remove(their_item)
+        self.add(their_item)
```

Comment:
👍 great job using the instance methods instead of appending and removing through the built-in list methods